components/organisms/ProfileHeader.tsx

```diff
@@ -0,0 +1,63 @@
+import { LeftChevronSVG } from "@ensdomains/thorin";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+export enum Tabs {
+  Profile = "profile",
+  Subdomains = "subdomains",
+}
+
+export const ProfileHeader = () => {
+  const router = useRouter();
+  const { name } = router.query; // Dynamic route parameter
```

Comment:
Can you attach the following doc. to [further](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes) define this?